### PR TITLE
README

### PR DESCRIPTION
**Pull request by malomarrec**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Cloudskiff
+Cloudskiff measures infrastucture as code coverage, and tracks cloud configuration drift.
+
+## Why ?
+
+Infrastructure as code is awesome, but there are too many moving parts: codebase, state file, actual cloud configuration. Things tend to drift.
+
+Drift can have multiple causes: from developers creating or updating infrastructure through the web console without telling anyone, to uncontrolled updates on the cloud provider side. Handling infrastructure drift vs the codebase can be challenging.
```
Comment (sjourdan) [inline README.md:8]: WDYT? "...to uncontrolled updates on the cloud provider side, _or undetected changes by the IaC tool_."

File: README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Cloudskiff
```
Comment (sjourdan) [inline README.md:1]: Should we replace Cloudskiff occurences by `driftctl`?

Comment (malomarrec) [reply README.md:1]: Pas bete

**Comment by malomarrec** [comment]
J'ai comprendu comment faire.
Better?
@sjourdan

**Review by sjourdan — APPROVED** [review]

**Pull request merged**